fdb-record-layer-core/src/main/proto/index_build.proto

```diff
@@ -31,6 +31,7 @@ message IndexBuildIndexingStamp {
     INVALID    = 0;   // let zero be invalid
     BY_RECORDS = 1;   // indexing by records - scanning all records in a record store
     BY_INDEX   = 2;   // indexing by a source index. When source index points to all possible records of the target index.
+    SCRUB_REPAIR =3;  // while readable, verify index validity. Repair if needed.
```

Comment:
```suggestion
    SCRUB_REPAIR = 3;  // while readable, verify index validity. Repair if needed.
```
Probably worth aligning the other values to the new position of the `=`, too

Reply:
Fixed.